lib/ansible/modules/database/postgresql/postgresql_idx.py

```diff
@@ -504,7 +527,10 @@ def main():
     schema = module.params["schema"]
 
     if concurrent and cascade:
-        module.fail_json(msg="Cuncurrent mode and cascade parameters are mutually exclusive")
+        module.fail_json(msg="Concurrent mode and cascade parameters are mutually exclusive")
+
+    if unique and (idxtype and idxtype != 'btree'):
+        module.fail_json(msg="Only btree currently supports unique indexes.")
```

Comment:
Just for consistency, let's remove dot at the end of the message. This is my final concern :)

Reply:
Done!